### ISSUE TITLE
Unable to use custom DB schema with enums

### ISSUE THREAD

**Issue by vitherain** [ArsenyYankovsky/typeorm-aurora-data-api-driver#167]
**Package versions and database engine type (please complete the following information):**
 - Database Engine: [postgres]
 - TypeORM Version: [0.3.10]
 - Driver Version [3.0.0]

**Describe the bug**
When some entity that uses some enum is in other-than-default database schema the SQL commands made upon this entity fail. It is because the cast parameter contains directly the name of the enum without DB schema prefix. PostgreSQL does not find such an enum because it is not specific enough.

**To Reproduce**

```
export enum StringEnum {
  ADMIN = 'a',
  EDITOR = 'e',
  MODERATOR = 'm',
  GHOST = 'g'
}

@Entity({ schema: 'foo' })
export class SpecificSchemaSimpleEnumEntity {
  @PrimaryColumn()
  public id!: number

  @Column({
    type: 'simple-enum',
    enum: StringEnum,
    default: StringEnum.GHOST,
  })
  public stringEnum!: StringEnum
}

const repository = connection.getRepository(SpecificSchemaSimpleEnumEntity)
const enumEntity = new SpecificSchemaSimpleEnumEntity()
enumEntity.id = 1
enumEntity.stringEnum = StringEnum.ADMIN
await repository.save(enumEntity)
```

Results in error `BadRequestException: Database error code: 0. Message: ERROR: type "specific_schema_simple_enum_entity_stringenum_enum" does not exist`

**Comment by vitherain** [comment]
I already created a fix for this error https://github.com/ArsenyYankovsky/typeorm-aurora-data-api-driver/pull/166.

**Comment by ArsenyYankovsky** [comment]
Thanks for reporting it and submitting a PR. 
I merged the PR, new version 3.0.1. was released with this fix.

@all-contributors please add @vitherain for code, bug.
